index.html

```diff
@@ -296,14 +169,16 @@ <h4>Ali Sahid</h4>
       <section class="partners" id="partners">
         <h3>Partners</h3>
         <div class="border"></div>
-        <div class="box-partner-A">
-          <img src="./assets/images/airbnb.svg" alt="Airbnb" />
-          <img src="./assets/images/google.svg" alt="Google" />
-          <img src="./assets/images/mozilla.svg" alt="mozilla" />
-        </div>
-        <div class="box-partner-B">
-          <img src="./assets/images/daum.svg" alt="daumkakao" />
-          <img src="./assets/images/naver.svg" alt="naver" />
+        <div class="row">
+          <div class="box-partner-A">
+            <img src="./assets/images/airbnb.svg" alt="Airbnb" />
+            <img src="./assets/images/google.svg" alt="Google" />
+            <img src="./assets/images/mozilla.svg" alt="mozilla" />
+          </div>
+          <div class="box-partner-B">
+            <img src="./assets/images/daum.svg" alt="daumkakao" />
+            <img src="./assets/images/naver.svg" alt="naver" />
+          </div>
         </div>
       </section>
       <footer>
```

Comment:
- [ ] Please review the `background-color` of the `footer` on mobile devices to ensure it aligns with the given design guidelines. It **should have a white background to maintain consistency throughout your project**. Kindly make the necessary changes to achieve a cohesive visual appearance. Consistency in design elements contributes to a polished and professional project. 🎨

   ### Your design:
   ![image](https://github.com/alyconr/DevOpsConf2023/assets/107952724/ce61e0c4-7ad2-4500-8560-9d2566b506c2)

   ### Behance design:
   ![image](https://github.com/alyconr/DevOpsConf2023/assets/107952724/5ef0ae57-1484-4c3b-b402-6184679eeebc)